lib/event_handler.ex

```diff
@@ -54,13 +54,14 @@ defmodule TelemetryInfluxDB.EventHandler do
         ) :: :ok
   def handle_event(event, measurements, metadata, config) do
     event_tags = Map.get(metadata, :tags, %{})
+    event_timestamp = Map.get(metadata, "_timestamp", DateTime.utc_now())
```

Comment:
Do we want a default value here? Would it be better to pass `nil` in the case where there is no `_timestamp` in the metadata to allow the old default behavior to happen instead?